go.mod

```diff
@@ -5,3 +5,5 @@ go 1.23.0
 toolchain go1.24.1
 
 require golang.org/x/oauth2 v0.30.0
+
+require rsc.io/qr v0.2.0
```

Comment:
Latest version 2018 https://pkg.go.dev/rsc.io/qr  but that's not necessarily a problem